### DISCUSSION TITLE
Welcome to Project-QR-Gen-Team-52-Frontend Discussions!

### DISCUSSION THREAD

**Discussion by biyiemmy** [zuri-training/Project-QR-Gen-Team-52-Frontend#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by biyiemmy** [comment]
Hi devs

**Reply by clappy203** [reply]
Hello @biyiemmy

**Reply by biyiemmy** [reply]
Im good, U?
